src/helper/response-handler.js

```diff
@@ -76,7 +76,10 @@ function wrapCallback(cb, options) {
       return cb(null, data.body || data);
     }
 
-    return cb(null, objectHelper.toCamelCase(data.body || data));
+    return cb(
+      null,
+      objectHelper.toCamelCase(data.body || data, [], { keepOriginal: options.keepOriginalCasing })
```

Comment:
If you're going to keep an options dict here, can't you just pass the whole object in instead of translating?

Reply:
it's a different options object, so no. I can use the same name if you think it's worth it. I used two different names because they're used in different contexts, but I'm fine with both options

Reply:
Don't change it on my account 😆 just something to consider. Does it need translation? Does that add cognitive overhead? I won't hold back an approval based on that.